builder/builder_utilities.livecodescript

```diff
@@ -555,42 +589,54 @@ private function builderMergExtDownloadedFilePath pMergExtVersion, pEdition
    return builderRepoFolder() & slash & "MergExt-" & pEdition & "-" & pMergExtVersion & ".zip"
 end builderMergExtDownloadedFilePath
 
-command builderMergExtEnsureZip pMergExtVersion, pEdition
-   local tZipPath
-   put builderMergExtDownloadedFilePath(pMergExtVersion, pEdition) into tZipPath
-   if there is not a file tZipPath then
-      local tUrl
-      put builderMergExtUrl(pMergExtVersion, pEdition) into tUrl
-      builderLog "message", "Downloading MergExt to" && tZipPath
-      get shell(merge("curl -o '[[tZipPath]]' '[[tUrl]]'"))
-      if the result is not zero then
-         builderLog "error", "Failed to download MergExt zip:" && it
-         throw "failure"
-      end if
-   end if
-end builderMergExtEnsureZip
+private command builderMergExtUnpack pMergExtVersion, pEdition
+   local tFolder
+   put builderUnpackFolder() & slash & "Ext" into tFolder
+   
+   local tZipFile
+   put builderMergExtDownloadedFilePath(pMergExtVersion, pEdition) into tZipFile
+   
+   local tURL
+   put builderMergExtUrl(pMergExtVersion, pEdition) into tURL
+   
+   builderEnsureZip tZipFile, tURL
+   
+   builderUnpackZip tZipFile, tFolder
+end builderMergExtUnpack
 
-command builderMergExtUnpack pMergExtVersion, pEdition
-   builderMergExtEnsureZip pMergExtVersion, pEdition
+////////////////////////////////////////////////////////////////////////////////
+
+function builderTSNetUrl pTSNetVersion
+   return "https://downloads.livecode.com/tsNet/tsNet_" & pTSNetVersion & ".zip"
```

Comment:
OK, I won't change the scripts or tests then. When you post an update the procedure will be to edit the version constant and post a PR for that after upload. Travis then runs the test to ensure the file is available. It also checks the version is a date but that really was only for mergExt because it is a whole bundle of individually versioned externals. If you want to use a version number that might be better and we can just drop the date check.